packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md

```diff
@@ -6,6 +6,22 @@ title: Okta Identity Engine API Products release notes 2022
 
 ## April
 
+### Weekly release 2022.04.1
+
+| Change | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Bugs fixed in 2022.04.1](#bugs-fixed-in-2022-04-1) | April 6, 2022 |
+
+#### Bugs fixed in 2022.04.1
+
+* An HTTP 500 internal server error sometimes occurred after saving an app instance. (OKTA-483001)
+
+* Active Directory email activation templates, translated for the Japanese, Korean, and Chinese languages, didn’t render correctly. (OKTA-469764)
+
+* The `initiate_login_uri` parameter was still validated for an IdP-initiated login with Okta flow when the Initiate login URI field in the Admin Console value was empty, and the OIDC app couldn’t be updated. (OKTA-447112)
```

Comment:
It's confusing. If the **Initiate login URI** field, in the Admin console, is left blank during editing, you could not update your app, because the `initiate_login_uri` parameter on an internal API was still being validated. The bug fix was to remove this validation. In retrospect, I think we could re-phrase this better, but I hesitate to because it's been signed-off by the dev, dev mgr, and our editing team. But let me look into it.

Reply:
Worked with SME to revise and simplify as follows: 

An OIDC app couldn’t be updated for an IdP-initiated login with Okta flow when the Initiate login URI field in the Admin Console was empty. (OKTA-447112)